tests/functional/activation_emails/README.md

```diff
@@ -0,0 +1,7 @@
+This folder will contain files containing email message summaries.
+
+The name of the file is the GMail message ID.
```

Comment:
Is this comment still relevant?

Reply:
yes, It's used in S3 Montitor file to put email contants like 'to' details and activation/reset link details.